src/Medulla/Kubernetes/Operator/Model/Database/ColumnSpec.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the Medulla Contributors under one or more agreements.
+// The Medulla Contributors licenses this file to you under the Apache 2.0 license.
+// See the LICENSE file in the project root for more information.
+
+namespace Operator.Model
```

Comment:
Namespace should be Medulla.Kubernetes.Operator.Model.Database

Use [file scoped namespaces](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces) instead of curly brace oriented ones